docs/timezone-draft.md

```diff
@@ -106,30 +106,30 @@ class Temporal.TimeZone {
 
   /** Given an absolute instant returns this time zone's corresponding
    * UTC offset, in nanoseconds (signed). */
-  getOffsetNanosecondsFor(absolute : Temporal.Absolute) : number;
+  getOffsetNanosecondsFor(instant : Temporal.Instant) : number;
 
   /** Given the calendar/wall-clock time, returns an array of 0, 1, or
    * 2 (or theoretically more, but not in any currently known time zone)
    * absolute times, that are possible points on the timeline
    * corresponding to it. In getAbsoluteFor(), one of these will be
    * selected, depending on the disambiguation option. */
-  getPossibleAbsolutesFor(dateTime : Temporal.DateTime) : array<Temporal.Absolute>;
+  getPossibleAbsolutesFor(dateTime : Temporal.DateTime) : array<Temporal.Instant>;
```

Comment:
It's not vital that these drafts are updated, they're outdated anyway; but if you're doing a search-and-replace then we may as well include these:

```suggestion
  getPossibleInstantsFor(dateTime : Temporal.DateTime) : array<Temporal.Instant>;
```